aep/0140.yaml

```
@@ -0,0 +1,17 @@
+rules:
+  aep-140-boolean-field-naming:
+    description: 'Boolean fields should omit the prefix "is".'
+    severity: warn
+    given: '$..[?(@.type == "boolean" && (@property.match(/^is/)))]~'
+    then:
+      field: '@key'
+      function: pattern
+      functionOptions:
+        match: '^is[A-Z].*$'
```

I think you want "notMatch" with a bit broader pattern:
```suggestion
        notMatch: '^[Ii][Ss]_?[A-Za-z0-9_-]*$'
```